src/app/ApplicationTemplate.Mobile/ApplicationTemplate.Mobile.csproj

```diff
@@ -90,9 +90,32 @@
 				<PackageReference Include="Mono.AotProfiler.Android" Version="6.0.0" />
 			</ItemGroup>
 			<ItemGroup>
+				<AndroidAotProfile Include="custom.aprof" />
 				<AndroidEnvironment Include="Android/environment.conf" />
 				<LinkDescription Include="Android/LinkerExclusions.xml" />
-				<AndroidAotProfile Include="custom.aprof" />
+			</ItemGroup>
+			<ItemGroup>
```

Comment:
I don't even know if this was necessary, it was working fine without it locally.

Reply:
Hum, I'm not sure for that one. Did you check the uno project to see what they're doing?

Reply:
Well it was a generated project with configuration for the icons and splash screen, but nothing was configured in this and it was working fine.